test/browser/test_webelem.py

```diff
@@ -360,189 +360,190 @@ def test_multiple_levels(self):
                  stubs.FakeChildrenFrame(second[2:4])]
         root = stubs.FakeChildrenFrame(first)
         children = webelem.get_child_frames(root)
-        self.assertEqual(len(children), 7)
-        self.assertIs(children[0], root)
+        assert len(children) == 7
+        assert children[0] is root
         for frame in [root] + first + second:
-            with self.subTest(frame=frame):
-                frame.childFrames.assert_called_once_with()
+            frame.childFrames.assert_called_once_with()
 
 
-class IsEditableTests(unittest.TestCase):
+class TestIsEditable:
 
     """Tests for is_editable."""
 
-    def setUp(self):
+    @pytest.yield_fixture(autouse=True)
+    def setup(self):
+        old_config = webelem.config
         webelem.config = None
+        yield
+        webelem.config = old_config
+
+    @pytest.yield_fixture
+    def stub_config(self, stubs):
+        config = stubs.ConfigStub({'input': {'insert-mode-on-plugins': True}})
```

Comment:
Why is `insert-mode-on-plugins` set here already, and then again later in the tests? Shouldn't this rather be something like ` config = stubs.ConfigStub({'input': {}})` instead?